napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -80,28 +93,83 @@ def _on_stdout_ready(self):
             self._output_widget.append(text)
 
     def install(self, pkg_list: Sequence[str]):
-        cmd = ['-m', 'pip', 'install', '--upgrade']
-        if running_as_bundled_app() and sys.platform.startswith('linux'):
+        if self._use_conda:
+            cmd = [
+                'install',
+                '-c',
+                'conda-forge',
+                '-y',
+                '--prefix',
+                self._conda_env_path,
+            ]
+        else:
+            cmd = ['-m', 'pip', 'install', '--upgrade']
+
+        if (
+            running_as_bundled_app()
+            and sys.platform.startswith('linux')
+            and not self._use_conda
+        ):
             cmd += [
                 '--no-warn-script-location',
                 '--prefix',
                 user_plugin_dir(),
             ]
+        print(' '.join(cmd + list(pkg_list)))
         self.process.setArguments(cmd + list(pkg_list))
         if self._output_widget:
             self._output_widget.clear()
         self.process.start()
 
     def uninstall(self, pkg_list: Sequence[str]):
-        args = ['-m', 'pip', 'uninstall', '-y']
+        if self._use_conda:
+            args = [
+                'remove',
+                '-y',
+                '--prefix',
+                self._conda_env_path,
+                '-c',
+                'conda-forge',
```

Comment:
Maybe also add channel support as a kwarg? Then you can do `installer.install('tensorflow', method='conda', channel='apple')` =)

Reply:
Fixed